src/System.Net.HttpListener/src/System/Net/Managed/ChunkedInputStream.cs

```diff
@@ -118,11 +118,19 @@ private void OnRead(IAsyncResult base_ares)
             try
             {
                 int nread = base.EndRead(base_ares);
+                if (nread == 0)
+                {
+                    _no_more_data = true;
+                    ares._count = rb.InitialCount - rb.Count;
```

Comment:
I'm confused about the meaning of some of the fields on the instance.  What does _count represent?  Is it something more than an input amount of the buffer to be read into?